devicetypes/smartthings/zigbee-window-shade.src/zigbee-window-shade.groovy

```diff
@@ -132,6 +132,7 @@ def getLastLevel() {
 }
 
 def levelEventHandler(currentLevel) {
+	def lastLevel = getLastLevel()
```

Comment:
I feel like we should name the variable something else just to be safe. But otherwise, I think it is ok.

Reply:
yeah, I'm sure that the local scope blows the groovy syntactic sugar away, but I agree with Donald here just so it's obvious what's going on.

`def priorLevel`? perhaps?

Reply:
Yeah, I thought of new name so hard, that I've forgotten this PR. Anyway, now it's updated.
@greens @dkirker